plugins/dependencies/src/main/kotlin/dependencies/Testing.kt

```diff
@@ -119,6 +124,21 @@ object Testing {
         }
     }
 
+    object KotestExtensions: IsNotADependency {
```

Comment:
This will work only with the Kotlin DSL and we also support Groovy.
Add `val kotestExtensions = KotestExtensions`

Reply:
Thanks, didn't realize the difference. I always use Kotlin DSL personally :)